docs/deployment/amazon-ecs.mdx

```diff
@@ -13,11 +13,60 @@ an S3 bucket before the `topaz` container starts up.
 The [topaz config new](/docs/command-line-interface/topaz-cli/configuration) CLI command can be used to generate
 your configuration file, then it should be uploaded to a bucket location that is accessible by the `init-config` container.
 
+:::caution Note
+If you intend to access the topaz console from a browser, you need to edit the generated configuration file and set the
+`fqdn` (fully-qualified domain name) field in all services to the DNS name of the topaz instance.
+For example, if you are running the topaz container with the `topaz.example.com` DNS name, you would set the `fqdn`
+field to `topaz.example.com:<port>`. Replace `<port>` with the port number that the service is running on. By default those are 8383 for
+the authorizer and 9393 for the directory services (reader, writer, model, importer, exporter).
+:::
+
 This example attaches an EBS volume to persist the configuration, certs, and directory database.  If it does not already exist
 you may need to create
 an [IAM ECS Infrastructure Role](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/infrastructure_IAM_role.html)
 to allow ECS to provision EBS volumes.
 
+## Certificates
+
+By default, topaz generates its own self-signed TLS certificates if none are provided. Those certificates are not
+trusted by browsers and prevent the topaz console from communicating with the service.
+There are two ways around it:
+
+### Trust the self-signed certificates
+
+You can download the generated certificate from the topaz instance using a browser or from the terminal using `openssl`:
```

Comment:
could we simply use "topaz certs trust"?

Reply:
No because topaz isn't running locally. The certs are generated inside the topaz container running in ECS.